project/src/main/java/com/todo/project/persistence/model/Ticket.java

```diff
@@ -1,24 +1,98 @@
 package com.todo.project.persistence.model;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.Table;
+import javax.persistence.*;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import lombok.AccessLevel;
 import lombok.Data;
 import lombok.Setter;
 
-@Data
```

Comment:
We can keep the Data annotation in order to save ourselves some boilerplate code like getters and setters.